app/services/project_search_service.rb

```diff
@@ -8,6 +8,10 @@ def search(query)
       return search_by_ukprn(query)
     end
 
+    if word_pattern(query)
```

Comment:
I wonder if this just becomes a case with a default? Does it look like a urn, do this, Does it look like a UKPRN, do this, otherwise assume it is a name?

Reply:
Done